### DISCUSSION TITLE
How to change install ?

### DISCUSSION THREAD

**Discussion by Daxcor69** [vmware-tanzu/velero#3612] [Community support Q&A] [answered]
here is the command when I installed the server portion.
```
velero install \
  --provider velero.io/aws \
  --bucket velero-backups \
  --plugins velero/velero-plugin-for-aws:v1.0.0,digitalocean/velero-plugin:v1.0.0 \
  --backup-location-config s3Url=https://nyc3.digitaloceanspaces.com,region=nyc3 \
  --use-volume-snapshots=false \
  --secret-file ./examples/cloud-credentials
```

Now I want to change a value. specifically  ```-use-volume-snapshots=false ``` to true.  Do run the command again or uninstall and reinstall?

I looked on the site, I can't find the info on how to configure the schedule. 

thanks.
Brad

**Comment by jenting** [comment, marked as answer]
Run the `velero install` again with the value you want to change.
